keylime/cmd/convert_runtime_policy.py

```diff
@@ -214,6 +211,7 @@ def main() -> None:
     policy_out = update_runtime_policy(
         policy, excludelist_path=args.excludelist, verification_keys=args.verification_keys
     )
+    ima.trim_runtime_policy(policy_out, False)
```

Comment:
Somehow I think that the "trimming" code should be part of the conversion, and should live here and not in the IMA module.

Reply:
It's called from many places.